docs(windows): add smartscreen knowlege base (#2510)
keymanapp/help.keyman.com: knowledge-base/kb0119.md

# Microsoft Defender SmartScreen

## Summary

> Due to changes in how certificate signing and trusted applications are handled, you *may* see the following SmartScreen notification when installing Keyman.

Reviewer: just minor tweaks:

```suggestion
Due to changes in how Microsoft handles certificate signing and trusted applications, you *may* see the following SmartScreen notification when installing Keyman for Windows or Keyman Developer.
```

 <img src="./assets/kb0119/smartscreen_initial.png" width="75%" alt="SmartScreen Notification "/>

## Installation

To continue installation, click **More info**. This will display the name of the Keyman installation file and the publisher name, Summer Institute of Linguistic (SIL) from the signing certificate.
It also adds a **Run anyway** button which will enable you to continue installation.
<img src="./assets/kb0119/smartscreen_keyman.png" width="75%" alt="SmartScreen Notification More Info "/>

> **What has changed**

Reviewer 2: ```suggestion
### What has changed
```

Previously, applications signed by Azure Artifact Signing (formerly Trusted Signing) meant that there was no SmartScreen prompts during installation. Microsoft Defender SmartScreen now relies also on application reputation, which is built over time based on factors determined by Microsoft.

> As a result, SmartScreen prompts may appear until the Keyman installer establishes sufficient reputation.

Reviewer 2: ```suggestion
As a result, SmartScreen prompts may appear until a new version of the Keyman installer establishes sufficient reputation.
```

This might make it clearer that this needs to happen for each new version.

> 

Reviewer 3: Perhaps add another line:

```suggestion

## How to know if the downloaded file is trustworthy

If you are presented with a SmartScreen warning, check the publisher in the "More Info" screen. If the publisher field shows the name "Summer Institute of Linguistics, Inc.", then that confirms that we built the installer and it can be trusted. See the image above for a representative example.

If any other name is shown in the Publisher field, do not run the installer, and please [contact us](/contact) for further assistance!
```
